src/Projection/ProjectorHelper.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Patchlevel\EventSourcing\Projection;
+
+use Patchlevel\EventSourcing\EventBus\Message;
+
+final class ProjectorHelper
```

Comment:
The naming is not hinting the user what the class is really doing. Can we get a better name here?

Reply:
Do you have a suggestion?

Reply:
Hm idk, creating a issue for that to think about it again before releasing `2.1.0`